### ISSUE TITLE
TMA support 

### ISSUE THREAD

**Issue by aayasin** [intel/PerfSpect#34]
I wonder how this tools select what nodes of the TMA tree to feature?? 

for example, it looks interested to support to Memory synchronization issues but selects just data sharing and False sharing. However,For such an analysis, Some nodes like contested access are missing .

also, it is unclear what is the the TMA metric version the tool has incorporated? Please include that in the output / logs. EDP is doing so for some years now.

**Comment by hilldani** [comment]
Perfspect follows the latest metrics and equations published at https://github.com/intel/perfmon. It does not include all metrics because it tries to remain lightweight and just include the most valuable metrics for diagnosing performance issues. If you have a metric you would like to include please let me know and we can consider adding it. Also you can add any additional custom metrics you want to the metric source files if you have a unique custom use case

**Comment by aayasin** [comment]
Yes, please let the tool report the TMA version included on Intel GitHub and by original EDP. 

The example missing node i gave is a real one. Otherwise the support for Performance issue being chased (In this case memory synchronization I assume) would be incomplete.
